src/Catty/DataModel/Bricks/Look/NextLookBrick.swift

```diff
@@ -20,37 +20,44 @@
  *  along with this program.  If not, see http://www.gnu.org/licenses/.
  */
 
-#import "PreviousLookBrick.h"
-#import "ObjectTableViewController.h"
-#import "Script.h"
-#import "Pocket_Code-Swift.h"
-
-@implementation PreviousLookBrick
-
-- (kBrickCategoryType)category
-{
-    return kLookBrick;
-}
-
-- (NSString *)brickTitle {
-    return @"";
-}
-
-- (NSString*)pathForLook:(Look*)look
-{
-    return [look pathForScene:self.script.object.scene];
+@objc(NextLookBrick)
+@objcMembers class NextLookBrick: Brick, BrickProtocol {
+    @objc var look: Look?
+
+    init(look: Look) {
+        self.look = look
+        super.init()
+    }
+
+    public required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override public required init() {
+        super.init()
+    }
+
+    func path(for look: Look?) -> String? {
+        look?.path(for: script.object.scene)
+    }
+
+    func brickTitle() -> String! {
```

Comment:
Is the function "brickTitle" really needed here? Since you set the brick title in the file NextLookBrickCell.swift.

Reply:
Thank you. You are right. I removed it.